docs/reference/glossary.md

```diff
@@ -169,11 +169,7 @@ A message contains information to be delivered to interested parties during exec
 
 ### Orchestration cluster
 
-An orchestration cluster includes Zeebe, Operate, Tasklist, Optimize, and Connectors. Previously [automation cluster](#automation-cluster).
-
-### Orchestration core
-
-An orchestration core or orchestration cluster core includes Zeebe, Operate, Tasklist, Optimize, and Identity.
+An orchestration cluster includes Zeebe, Operate, and Tasklist, but may include optional components like Optimize and Connectors. Previously [automation cluster](#automation-cluster).
```

Comment:
Considering this is preparing for 8.8 architecture, I would frame it more something like this

```suggestion
An orchestration cluster includes the Zeebe, Operate, Tasklist, and an orchestration cluster level Identity. With these, it can autonomously execute process workloads, serve the relevant Frontend UIs, ensure authentication and authorization, and serve the gRPC and REST APIs. Previously [automation cluster](#automation-cluster).
```

Reply:
I made small tweaks and will accept this suggestion.